openquake/sep/classes.py

```diff
@@ -561,12 +561,22 @@ def compute(self, mag, imt_gmf, sites):
         return out
 
 
+def get_session(model):
+    """
```

Comment:
I would suggest that we add a short comment here explaining why we are deliberately disabling ONNX multi-threading.   Something like:

The engine is already using multiple threads, so we must use only one thread (per OQ worker) for ONNX operations in order to avoid creating more threads that we have CPUs.